samples/transform_feedback_instanced.html

```diff
@@ -0,0 +1,380 @@
+            gl.vertexAttribPointer(POSITION_LOCATION, 2, gl.FLOAT, false, 0, 0);
+            gl.enableVertexAttribArray(POSITION_LOCATION);
+
+            vertexBuffers[va][COLOR_LOCATION] = gl.createBuffer();
+            gl.bindBuffer(gl.ARRAY_BUFFER, vertexBuffers[va][COLOR_LOCATION]);
+            gl.bufferData(gl.ARRAY_BUFFER, instanceColors, gl.STATIC_DRAW);
+            gl.vertexAttribPointer(COLOR_LOCATION, 3, gl.FLOAT, false, 0, 0);
+            gl.enableVertexAttribArray(COLOR_LOCATION);
+            gl.vertexAttribDivisor(COLOR_LOCATION, 1); // attribute used once per instance
+
+            gl.bindVertexArray(null);
+        }
+
+        // -- Init TransformFeedback
+        var transformFeedback = gl.createTransformFeedback();
+
+        render();
+
+        function initPrograms() {
+
+            // Setup program for transform feedback shaders
+            function createShader(gl, source, type) {
+                var shader = gl.createShader(type);
+                gl.shaderSource(shader, source);
+                gl.compileShader(shader);
+                return shader;
+            }
+
+            var vshaderTransform = createShader(gl, getShaderSource('vs-emit'), gl.VERTEX_SHADER);
+            var fshaderTransform = createShader(gl, getShaderSource('fs-emit'), gl.FRAGMENT_SHADER);
+
+            var programTransform = gl.createProgram();
+            gl.attachShader(programTransform, vshaderTransform);
+            gl.deleteShader(vshaderTransform);
+            gl.attachShader(programTransform, fshaderTransform);
+            gl.deleteShader(fshaderTransform);
+
+            var varyings = ['v_offset', 'v_rotation'];
+            gl.transformFeedbackVaryings(programTransform, varyings, gl.SEPARATE_ATTRIBS);
+            gl.linkProgram(programTransform);
+
+            // check
+            var log = gl.getProgramInfoLog(programTransform);
+            if (log) {
+                console.log(log);
+            }
+
+            log = gl.getShaderInfoLog(vshaderTransform);
+            if (log) {
+                console.log(log);
+            }
+
+            // Setup program for draw shader
+            var programDraw = createProgram(gl, getShaderSource('vs-draw'), getShaderSource('fs-draw'));
+
+            var programs = [programTransform, programDraw];
+            return programs;
+        }
+
+        function transform() {
+            var programTransform = programs[PROGRAM_TRANSFORM];
+
+            // Toggle source and destination VBO
+            var sourceVBO = vertexBuffers[currentSourceIdx];
+            var sourceVAO = vertexArrays[currentSourceIdx];
+
+            var destinationVBO = vertexBuffers[(currentSourceIdx + 1) % 2];
+
+            gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, transformFeedback);
+
+            gl.useProgram(programTransform);
+
+            gl.bindVertexArray(sourceVAO);
+
+            // need to set to 0 when doing transform feedback

```

Comment:
For clarity, it might be good to change the comment to "Attributes per-vertex when doing transform feedback".